webapp/components/admin_console/text_setting.jsx

```diff
@@ -67,7 +66,6 @@ export default class TextSetting extends React.Component {
                     id={this.props.id}
                     className='form-control'
                     rows='5'
-                    maxLength='1024'
```

Comment:
Maybe keep in `maxLength={this.props.maxLength}`? That way, people can set a max length for textareas if they want it